lib/scholar/cluster/affinity_propagation.ex

```diff
@@ -56,6 +58,14 @@ defmodule Scholar.Cluster.AffinityPropagation do
       doc: ~S"""
       If `true`, the learning loop is unrolled.
       """
+    ],
+    convergence_iter: [
```

Comment:
We only use full `iteration*`. `converge_after` is cool because it indicates the purpose of this parameter but doesn't have a simple connotation with the number of iterations. But I guess the first one is more important so I'm okay with this name

Reply:
Converge_after is shorter too :)